docs/manage-security-verification-for-wog-account.md

```diff
@@ -0,0 +1,33 @@
+# Set up security verification for Whole of Government (WOG) account
+
+We know that public officers can use their TechPass account to access Singapore Tech Stack (SGTS) services and tools. However, if public officers want to access SGTS services and tools from their Government Managed Device (GMD) using TechPass, they must first set up security verification for their WOG account. This adds an additional level of security layer when they access these resources from their GMD.
```

Comment:
Suggestion: 
"Public officers can use their TechPass account to access Singapore Tech Stack (SGTS) services and tools across Non-SE GSIB and/or GMD devices. ........... This additional level of security layer is required when they access these resources from their GMD"

Reply:
Done.